bigbluebutton-html5/imports/ui/services/audio-manager/index.js

```diff
@@ -409,11 +409,15 @@ class AudioManager {
     return this._userData;
   }
 
-  notify(message, error = false) {
+  notify(message, error = false, icon) {
```

Comment:
How about
```
 notify(message, error = false, icon = 'unmute') {
    let audioIcon = this.isListenOnly ? 'listen' : icon;
```

instead of
```
 notify(message, error = false, icon) {
    let _icon = 'unmute';
    _icon = icon || _icon;
    _icon = this.isListenOnly ? 'listen' : _icon;
```
?